app/models/user.rb

```diff
@@ -3,7 +3,8 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  has_many :metrics
+  
+  has_many :metrics, through: :features
```

Comment:
@dcox Oh this is good but surprising, I wrongly assumed there'd be an error as a has-many-through is usually paired with a has-many that is named in the `through: ` option.

In this case, I'd expected it to need this association:
```diff
class User ...
  ...
+ has_many :features
  has_many :metrics, through: :features
  ...
```